content/docs/esc/other-integrations/github-actions.md

```diff
@@ -0,0 +1,221 @@
+---
+title_tag: Integrate with GitHub Actions | Pulumi ESC
+title: GitHub Actions
+h1: "Pulumi ESC: Integrate with GitHub Actions"
+meta_desc: This page provides an overview of how to use Pulumi ESC with GitHub Actions.
+weight: 2
+menu:
+  pulumiesc:
+    parent: esc-other-integrations
+    identifier: esc-other-integrations-github-actions
+---
+
+## Overview
+
+Pulumi ESC integrates with [GitHub Actions](https://docs.github.com/en/actions) (GHA) to help developers automatically manage configuration and secrets when running workflows. Additionally, Pulumi ESC works with the [Pulumi CLI Action](https://github.com/marketplace/actions/pulumi-cli-action#pulumi-github-actions) to provide secrets to workflow jobs and steps, such as AWS Credentials.
```

Comment:
I'm not sure we should abbreviate GitHub actions; their docs seem not to: https://docs.github.com/en/actions

Reply:
Good point, I'll spell it out going forward!